html/profile.html

```diff
@@ -10,9 +10,91 @@
 </head>
 
 <body>
-  <h1 class="title-of-page">Setting up Profile.</h1>
 
-  <script src="../script/profile.js"></script>
+  <div class="app">
+
+    <div class="app__body">
+
+      <div class="sidebar">
+
+        <div class="sidebar__header">
+          <img src="../image/img_avatar.png" alt="avtar" class="avatar">
```

Comment:
spelling of avatar is misspelled in `alt` attribute

Reply:
Corrected it